p2p/peermanager.go

```diff
@@ -695,7 +695,6 @@ func (m *PeerManager) Ready(peerID NodeID) error {
 			Status: PeerStatusUp,
 		})
 	}
-	return nil
```

Comment:
Is there any value in returning a boolean to catch if the peer is actually not connected?

Reply:
I don't know how we would, in the peer manager, know this?

I think this API is sufficiently internal that we don't really need to worry about changing it later if we need/want to.

Reply:
Yeah so I ask this question because in `routePeer` we call `Ready()` on the new peerID but if this for some reason is not in the connected map then the peer manager won't add this to the ready map yet `routePeer` will assume it has. It might not actually be possible to happen or it might be inconsequential but it's just what I was thinking about when I asked it

Reply:
Ah! That makes sense. I'm not sure that it matters (or at least, the old code wasn't doing this check,) but it might be good to add this to the peer manager in the future.